challenges/functions.js

```diff
@@ -7,12 +7,34 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+const consume = function(param1, param2, cb) {
+  return cb(param1, param2);
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+// -------------------------------ADDITION CB-------------------------------------
```

Comment:
All of these below are the callback functions, so they won't take in a cb as a parameter. Line 11 above is going to invoke whichever function you call on -- so the parameters there in line 11 need to match whatever function you are calling.